Library/Homebrew/api/analytics.rb

```diff
@@ -12,7 +12,7 @@ def analytics_api_path
         end
         alias generic_analytics_api_path analytics_api_path
 
-        sig { params(category: String, days: T.any(Integer, String)).returns(Hash) }
+        sig { params(category: String, days: T.any(Integer, String)).returns(T::Hash[String, Integer]) }
```

Comment:
```suggestion
        sig { params(category: String, days: T.any(Integer, String)).returns(T::Hash[String, T.untyped]) }
```

JSON values are generally untypable, unless you know the shape of the output. Since this spans multiple endpoints, let's just leave them untyped. (CI failures are due to non-Integer values in the output).